dotnet/src/IntegrationTests/Connectors/Milvus/MilvusMemoryStoreTests.cs

```diff
@@ -0,0 +1,308 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Connectors.Memory.Milvus;
+using Microsoft.SemanticKernel.Memory;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Milvus;
+
+public class MilvusMemoryStoreTests : IAsyncLifetime
+{
+    private const string MilvusHost = "127.0.0.1";
```

Comment:
I'm getting errors for these tests, expected?

> Milvus.Client.MilvusException : ErrorCode: IndexNotExist Reason: there is no index on collection: 444404971368874468 with the index name: _default_idx

Reply:
2.3 version of milvus not supported. A readme with instructions on setting up milvus locally would be useful, along with a list of known supported versions.